detectron2/utils/testing.py

```diff
@@ -1,7 +1,10 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
 import io
 import numpy as np
+import os
+import unittest
 import torch
+from packaging import version
```

Comment:
A guess that this import might be causing some inconsistencies. 
`packaging` is a third-party library so it should be declared in `setup.cfg` as a thirdparty, so isort knows it is a thirdparty.
It should also be declared in setup.py.

isort uses what's importable in the environment to determine whether something is third-party. - so this behavior can be inconsistent across environemtn

Reply:
Thank you, I will try that

Reply:
@ppwwyyxx Could you approve one more run? hoppefully your suggestion does the trick :)